staging/src/k8s.io/legacy-cloud-providers/gce/gce_addresses.go

```diff
@@ -117,16 +117,6 @@ func (g *Cloud) GetRegionAddress(name, region string) (*compute.Address, error)
 	return v, mc.Observe(err)
 }
 
-// GetAlphaRegionAddress returns the Alpha, regional address by name.
```

Comment:
We aren't allowing alpha addresses to be gotten but they can still be created? (ReserveAlphaRegionAddress() above)

Reply:
@cheftako thanks for spotting this ! I pushed a new commit that fixes it.